pkg/agent/controller/networkpolicy/reconciler.go

```diff
@@ -873,22 +873,22 @@ func (r *reconciler) GetRuleByFlowID(ruleFlowID uint32) (*types.PolicyRule, bool
 func (r *reconciler) getOFPorts(members v1beta2.GroupMemberSet) sets.Int32 {
 	ofPorts := sets.NewInt32()
 	for _, m := range members {
-		var entityName, ns string
 		var ifaces []*interfacestore.InterfaceConfig
+		var name, ns string
```

Comment:
why changing this?  entityName represents this is the name of an entity, either a Pod or an ExternalEntity.